sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -147,31 +146,53 @@ public class AppConfigurationTestEnvironment : TestEnvironment
 }
 ```
 
+### Running live tests serially
+
+By default, NUnit does not run tests within each assembly in parallel, but this be [configured](https://docs.nunit.org/articles/nunit/technical-notes/usage/Framework-Parallel-Test-Execution.html).
+Especially for unit tests, this is often desirable; however, live and [recorded tests](#recorded-tests) may run into some issues. Thus, by default, the `RecordedTestBase` described below is attributed
+as `[NonParallelizable]`.
+
+However, when projects are built and tested in CIs, all projects are testing in parallel. This means, for example, you can have two or more assemblies running tests such as one backing up or restoring
```

Comment:
```suggestion
However, when projects are built and tested in CI pipelines, all projects are testing in parallel. This means, for example, you can have two or more assemblies running tests such as one backing up or restoring
```

Reply:
CIs - Continuous Integration - already imply some form of pipeline, but I don't see harm in taking this if it helps clear it up.